<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NameSelecting</class>
 <widget class="QMainWindow" name="NameSelecting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_names">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gloucester MT Extra Condensed</family>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Input names:</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>337</width>
      <height>204</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="first_player">
             <property name="font">
              <font>
               <family>Gloucester MT Extra Condensed</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>First player`s name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="second_player">
             <property name="font">
              <font>
               <family>Gloucester MT Extra Condensed</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Second player`s name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="third_player">
             <property name="font">
              <font>
               <family>Gloucester MT Extra Condensed</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Third player`s name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="fourth_player">
             <property name="font">
              <font>
               <family>Gloucester MT Extra Condensed</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Fourth player`s name</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLineEdit" name="first_name"/>
           </item>
           <item>
            <widget class="QLineEdit" name="second_name"/>
           </item>
           <item>
            <widget class="QLineEdit" name="third_name"/>
           </item>
           <item>
            <widget class="QLineEdit" name="fourth_name"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="ok_btn">
         <property name="text">
          <string>Ok</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
